app/assets/javascripts/modules/webchat.js

```diff
@@ -0,0 +1,151 @@
+(function (global) {
```

Comment:
This doesn't match the [module pattern](https://github.com/alphagov/govuk_frontend_toolkit/blob/master/docs/javascript.md#modules). Could it live in the parent directory or a different subdir? Then `//= require_tree ./modules` continues to work as before.

eg, it's not in the form:

```js
GOVUK.Modules.TrackShareButtonClicks = function () {
    this.start = function (element) {
```